listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -34,7 +37,7 @@ def get_playing_now(self, user_id):
                 Listen object which is the currently playing song of the user
 
         """
-        data = self.redis.get(self.ns + 'playing_now:{}'.format(user_id))
+        data = cache.get(self.PLAYING_NOW_KEY + str(user_id))
```

Comment:
in `__init__` you create `self.redis`, but here you just use the cache. Later in this file (check_connection) you use `cache._r` instead of `self.redis`. Could be made more consistent.